### DISCUSSION TITLE
Problem with Chrome Plugin

### DISCUSSION THREAD

**Discussion by taminelli** [dani-garcia/vaultwarden#3257] [Q&A]
I have reinstall my pc. After that I can no longer login with chrome plugin.
URL is correct. [https://bitwarden.domain.com:5152 1](https://bitwarden.domain.com:5152/)

Message: Cannot read properties of null (reading ‘iterations)

Login on website is possible.

**Comment by befabry** [comment]
Hello,

Same error here. I re-installed Linux today and can login only through the web, can't log in through Firefox or Chrome extensions

**Reply by tessus** [reply]
Downgrade the plugin to `2023.1.0`

**Comment by BlackDex** [comment]
Duplicate of #3259
